src/utils/Meta.js

```diff
@@ -1,60 +1,88 @@
 import _ from 'lodash'
+import * as stardust from 'stardust'
+
+const LIBRARIES = {
+  semanticUI: 'Semantic UI',
+  stardust: 'Stardust',
+}
+
+const TYPES = {
+  addon: 'addon',
+  global: 'global',
+  collection: 'collection',
+  element: 'element',
+  view: 'view',
+  module: 'module',
+}
+
+const LIBRARY_VALUES = _.values(LIBRARIES)
+const TYPE_VALUES = _.values(TYPES)
+const NAME_VALUES = _.keys(stardust)
+
+/**
+ * Determine if an object qualifies as a META object.
+ * It must have the required keys and valid values.
+ * @private
+ * @param {Object} _meta A proposed Stardust _meta object.
+ * @returns {Boolean}
+ */
+const isMeta = (_meta) => (
+  _.includes(LIBRARY_VALUES, _.get(_meta, 'library')) &&
+  _.includes(TYPE_VALUES, _.get(_meta, 'type')) &&
+  _.includes(NAME_VALUES, _.get(_meta, 'name'))
+)
```

Comment:
We can now validate `_meta` objects.  Meta is a static component prop, we need this below to identify / find the meta object from many possible sources.